src/suggestEnvironmentVars.ts

```diff
@@ -40,6 +53,7 @@ function createSuggestion(
   let item = new vscode.CompletionItem(env, vscode.CompletionItemKind.Variable);
   item.detail = `${envVal} (src: ${envSrc})`;
   item.range = new vscode.Range(position, position);
+  item.filterText = "*";
```

Comment:
Again - you should explain the *why* and reasoning as a comment. This is quite tricky, and the knowledge/learning shouldn't be lost.